cmd/podman/system/reset.go

```diff
@@ -61,7 +61,9 @@ func reset(cmd *cobra.Command, args []string) {
         - all pods
         - all images
         - all networks
-        - all build cache`)
+        - all build cache
+        - all machines`)
+
```

Comment:
Is there an md file that should have this tweak too?  I'm not seeing a changed one here on a quick scan.